casper/src/main/resources/PoS.rhox

```diff
@@ -431,60 +431,50 @@ in {
               new commitRewards in {
                 // Private method which transfers all of a validator's rewards + bond to the Coop vault.
                 // Slash expects (Boolean, Either[Nil, String]) on returnCh.
-                contract PoS(@"slash", @deployerId, @blockHash, @sysAuthToken, returnCh) = {
```

Comment:
Do you have an idea why these two params removed were introduced here previously?

Reply:
I don't know. But I can add this to another chapter to my book _How to not build the blockchain_. :smile:

I'm guessing it's to validate slashing in replay in some way but that was not implemented at all.

What is even worse, if invalid blocks are not set correctly, node will use it own public key and slash itself. (WTF?)